modules/nw-ovn-kubernetes-session-affinity.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+:_content-type: CONCEPT
+[id="nw-ovn-kubernetes-session-affinity_{context}"]
+= Session affinity
+Session affinity applies services to `objects`. For example, a Service's IP:Port can use a proxy to connect to a designated backend. You can set session affinity to connect a specified client to the same Pod each time. For more information, including how to set session affinity based on a client's IP address, refer to link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
+
+[discrete]
+[id="nw-ovn-kubernetes-session-affinity-stickyness-timeout_{context}"]
+== Stickyness timeout for session affinity
+The OVN-Kubernetes network plugin for {product-title} calculates the stickyness timeout for a `session` from a client based on the last packet. For example, if you run a curl 10 times, the stickyness timer starts from 10th packet not 1st. As a result if the client is continuously contacting the service, then the session will never timeout. The timeout kicks in when the service has not received a packet for `timeoutSeconds`.
```

Comment:
Link to https://kubernetes.io/docs/reference/networking/virtual-ips/#session-stickiness-timeout at the end when you say `timeoutSeconds` so that its clear what that value is and how that can be set.